spec/features/distribution_spec.rb

```diff
@@ -53,6 +53,27 @@
     expect(page).to have_content 13
   end
 
+  scenario "User creates a distribution from a donation then tries to make the quantity too big" do
+    @donation = create :donation, :with_items
+
+    visit @url_prefix + "/donations/#{@donation.id}"
+    click_on "Create distribution"
+    select @partner.name, from: "Partner"
+    click_button "Preview Distribution"
+    expect(page).to have_content "Distribution Manifest for"
+    click_button "Confirm Distribution"
+    expect(page.find(".alert-info")).to have_content "reated"
+    expect(Distribution.first.line_items.count).to eq 1
+
+    first(".btn", text: "Edit").click
+    first(".numeric").set 999_999
+    click_on "Update Distribution"
+    expect(page).to have_no_content "Distribution updated!"
+    expect(page).to have_content "Distribution could not be updated!"
+    expect(page).to have_no_content 999_999
+    expect(Distribution.first.line_items.count).to eq 1
```

Comment:
This shouldn't change even if it did update the quantity to 999,999 -- did you mean to test for something else?

Reply:
I tested in the first one that it didn't "split" into two items so I decided to keep it here just in case.